apps/mobile/src/features/browser/browser/browser-sheet.tsx

```diff
@@ -66,7 +66,7 @@ export function BrowserSheet() {
     <Sheet
       ref={browserSheetRef}
       snapPointVariant="fullHeightWithoutNotch"
-      shouldHaveContainer={false}
+      shouldHaveContainer={true}
```

Comment:
Not a big deal but I think you can just specify it as `shouldHaveContainer` when its positive (no `={true}`